knowledge_base/bundle_variables/README.md

```diff
@@ -0,0 +1,65 @@
+# bundle_variables
```

Comment:
I think a real title would be nice. Given it's a KB topic, how about "How to validate bundle variables"?